yarn.lock

```diff
@@ -1460,6 +1470,40 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@sandbox-smart-contracts/example_hardhat@workspace:packages/example_hardhat":
```

Comment:
not sure about having an actual reference to the example package in the lock

Reply:
All added packages seem to get referenced in `yarn.lock`